examples/todoapp/pubspec.yaml

```diff
@@ -28,8 +29,9 @@ dependencies:
   json_annotation: ^4.9.0
   jwt_decoder: ^2.0.1
   provider: ^6.0.5
+  realm: ^3.4.1
   retrofit: ^4.0.2
-  rx_bloc: ^5.0.1
+  rx_bloc: ^5.0.3
```

Comment:
Be aware that soon the rx_bloc ecosystem will introduce a breaking change and the versions needs to be updgraded. Also make sure the dependency overrides are utilised